api/api-elastic-profile-v2/src/main/java/com/thoughtworks/go/apiv2/elasticprofile/ElasticProfileControllerV2.java

```diff
@@ -121,8 +121,8 @@ public String create(Request request, Response response) {
     private void populatePluginIdFromReferencedClusterProfile(ElasticProfile elasticProfileToCreate) {
         ClusterProfile associatedClusterProfile = clusterProfilesService.findProfile(elasticProfileToCreate.getClusterProfileId());
         haltIfSpecifiedClusterProfileDoesntExists(associatedClusterProfile, elasticProfileToCreate);
-        elasticProfileToCreate.setPluginId(associatedClusterProfile.getPluginId());
-        elasticProfileToCreate.encryptSecureConfigurations();
+//        elasticProfileToCreate.setPluginId(associatedClusterProfile.getPluginId());
```

Comment:
Remove comments.
No need of explicitly setting pluginId on elastic agent profile.